packages/block-library/src/term-name/block.json

```diff
@@ -0,0 +1,70 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 3,
+	"__experimental": true,
+	"name": "core/term-name",
+	"title": "Term Name",
+	"category": "theme",
+	"description": "Displays the name of a taxonomy term.",
+	"textdomain": "default",
+	"usesContext": [ "termId", "taxonomy" ],
+	"attributes": {
+		"textAlign": {
+			"type": "string"
+		},
+		"tagName": {
+			"type": "string",
+			"default": "div"
+		},
+		"isLink": {
+			"type": "boolean",
+			"default": false
+		}
+	},
+	"supports": {
+		"align": [ "wide", "full" ],
+		"html": false,
+		"color": {
+			"gradients": true,
+			"link": true,
+			"__experimentalDefaultControls": {
+				"background": true,
+				"text": true,
+				"link": true
+			}
+		},
+		"spacing": {
+			"margin": true,
+			"padding": true
+		},
+		"typography": {
+			"fontSize": true,
+			"lineHeight": true,
+			"__experimentalFontFamily": true,
+			"__experimentalFontWeight": true,
+			"__experimentalFontStyle": true,
+			"__experimentalTextTransform": true,
+			"__experimentalTextDecoration": true,
+			"__experimentalLetterSpacing": true,
+			"__experimentalDefaultControls": {
+				"fontSize": true
+			}
+		},
+		"interactivity": {
+			"clientNavigation": true
+		},
+		"__experimentalBorder": {
+			"radius": true,
+			"color": true,
+			"width": true,
+			"style": true,
+			"__experimentalDefaultControls": {
+				"radius": true,
```

Comment:
NIt: I think we can reduce the default controls like this `borderRadius` and probably `margin`.

Reply:
It makes sense to reduce the controls here. I've removed the radius and margin controls in https://github.com/WordPress/gutenberg/pull/72129/commits/a4f201c1afcef3953801525d6c294ca1ab4b1c53.